spring-cloud-kubernetes-controllers/spring-cloud-kubernetes-configuration-watcher/src/main/java/org/springframework/cloud/kubernetes/configuration/watcher/ConfigurationWatcherAutoConfiguration.java

```diff
@@ -45,6 +45,7 @@
  */
 @Configuration(proxyBeanMethods = false)
 @EnableConfigurationProperties({ ConfigurationWatcherConfigurationProperties.class })
+@Import({ ConfigurationWatcherAutoConfiguration.RefreshTriggerConfiguration.class })
```

Comment:
instead of passing arguments for both bus and http based triggering, pass implementations of `BusRefreshTrigger` and `HttpRefreshTrigger`